nodejs/week2/app.js

```diff
@@ -0,0 +1,64 @@
+const express = require('express');
+const data = require('./documents.json');
+const app = express();
+const port = process.env.PORT || 3000;
+
+// Support parsing JSON requests
+app.use(express.json());
+
+app.get('/', (req, res) => {
+    res.send('This is a search engine');
+});
+
+app.get('/search', (req, res) => {
+    const queryValue = req.query.q;
+    if (queryValue) {
+        const matchingQuerry = data.filter(item => {
```

Comment:
Double-check for spelling, I know it is very minor but be careful. And also the name should be descriptive.

Reply:
I made the name more descriptive.

Thank you for your time!